### PR TITLE
New Airport ~ Manchester Airport w/ new airlines

### PR DESCRIPTION
**Pull request by eliuuk**
Manchester Airport is the 3rd busiest airport in the UK, and here it is!
- All SIDS (ASMIM, DESIG, EKLAD, KUXEM, LISTO, MONTY, Pole Hill,  SONEX and SANBA)
- All STARS (DALEY, ROSUN, MIRSI and DAYNE)
- Some additional airlines:
   Brussels Airlines,
   Monarch,
   Jet2 and
   Flybe
- Some additional aircraft:
  Avro RJ-100
  Saab 2000,
  Saab 340, and the
  de Havilland Canada DHC-6 Twin Otter

Brussels was also edited to include the national airline.

### CODE REVIEW

File: assets/airports/egcc.json [hunk 1]
```diff
@@ -0,0 +1,582 @@
+ {
+  "name": "Manchester Airport",
+  "level": "medium",
+  "radio": {
+    "twr": "Manchester Delivery",
```
Comment (erikquinn) [inline assets/airports/egcc.json]: Are you certain this is called "Manchester Delivery"? The one who tells them cleared to land and cleared for takeoff?

File: assets/airports/egcc.json [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+		"TNT":		["N53d03m14.000", "W001d40m12.000"],
+		"UPTON":	["N53d35m13.000", "W001d18m03.000"],
+		"WAL":		["N53d23m05.000", "W003d08m04.000"],
+		"WAL33":	["N53d29m51.000", "W002d13m77.000"],
+		"XOBRO":	["N53d28m17.000", "W002d25m18.000"],
+		"XUMAT":	["N53d27m95.000", "W002d28m38.000"],
+		"_05L":		["N53d20m51.200", "W002d17m15.950"],
+		"_05R":		["N53d19m55.100", "W002d18m38.380"],
+		"_23L":		["N53d21m40.750", "W002d15m33.410"],
+		"_23R":		["N53d20m53.350", "W002d16m37.950"],
```
Comment (erikquinn) [inline assets/airports/egcc.json]: These two fixes are backwards. It looks like `"_23L"` and `"_23R"` should be switched.

File: assets/airports/egcc.json (hunk 1 above)
Comment (eliuuk) [inline assets/airports/egcc.json]: It's not Manchester Delivery, thanks for reminding me to change that! The charts say "TWR : MANCHESTER DELIVERY" whereas the actual tower is Manchester Tower, which is the landing and takeoff clearance.

**Comment by eliuuk** [comment]
Again, thanks for seeing that - got mixed up again with those two!

**Comment by eliuuk** [comment]
'aircraft taxi' seems to have an issue. Specifying a runway through 'aircraft taxi runway' only seems to work.

Error:
http://prntscr.com/b6e316

**Comment by erikquinn** [comment]
interesting...

**Comment by eliuuk** [comment]
05L and 05R SIDS do not work, 23L and 23R SIDS or SIDS for all runways do work.
-> ASMIM
-> DESIG
http://prntscr.com/b6enu2

**Comment by erikquinn** [comment]
Depends on resolution to #519.

**Comment by erikquinn** [comment]
@indianbhaji if you refresh until the game chooses to use the 5's, then they will work and the 23's will not. Currently, aircraft aren't smart enough to file for SIDs that are valid from the runways currently in use, or smart enough to realize that they can't fly the SID if they do depart on that runway. The latter is causing a crash, which I am working on fixing now. The former will happen eventually, but not very soon.

**Comment by eliuuk** [comment]
Alright, thanks, for the time being I shall seek to remove those SIDS and reinstate them when this issue is fixed.

**Comment by erikquinn** [comment]
@indianbhaji Noooo, leave them in there! I've just about got it fixed. haha

**Comment by eliuuk** [comment]
aha, I'll put them back!

**Comment by eliuuk** [comment]
Airport finished. I would go further with a custom airspace, but the one that exists in real life seems to be quite confusing.

**Comment by erikquinn** [comment]
Merged via d78102ecfdad7cfb1e125cfd4ca037118cca7075. Closing.

**Pull request closed without merging**